manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -316,26 +344,26 @@ func (r *FybrikApplicationReconciler) reconcile(applicationContext *api.FybrikAp
 		if err := r.deleteExternalResources(applicationContext); err != nil {
 			return ctrl.Result{}, err
 		}
-		r.Log.V(0).Info("no blueprint will be generated since no datasets are specified")
+		r.Log.Info().Str(logging.FYBRIKAPPUUID, uuid).Msg("No plotter will be generated since no datasets are specified")
 		return ctrl.Result{}, nil
 	}
 
 	// create a list of requirements for creating a data flow (actions, interface to app, data format) per a single data set
 	// workload cluster is common for all datasets in the given application
 	workloadCluster, err := r.GetWorkloadCluster(applicationContext)
 	if err != nil {
-		r.Log.V(0).Info("could not determine in which cluster the workload runs: " + err.Error())
+		r.Log.Fatal().Err(err).Bool(logging.FORUSER, true).Bool(logging.AUDIT, true).Str(logging.FYBRIKAPPUUID, uuid).Str(logging.ACTION, logging.CREATE).Msg("Could not determine in which cluster the workload runs")
```

Comment:
why logging.CREATE? It is not a failure to create a resource.

Reply:
It's a failure while creating the data plane